src/main/java/io/meeds/qa/ui/pages/page/factory/settings/SettingsPage.java

```diff
@@ -9,27 +9,19 @@
 import net.serenitybdd.core.annotations.findby.FindBy;
```

Comment:
please avoid modifying the format in this file if no real changes are introduced